ESP8266Interface.cpp

```diff
@@ -425,6 +425,10 @@ int ESP8266Interface::socket_sendto(void *handle, const SocketAddress &addr, con
 {
     struct esp8266_socket *socket = (struct esp8266_socket *)handle;
 
+    if((strcmp(addr.get_ip_address(), "0.0.0.0") == 0) | !addr.get_port())  {
```

Comment:
Should this check use logical OR instead of bitwise OR?

Reply:
Thanks, that's true